hack/dind-cluster.sh

```diff
@@ -67,14 +67,26 @@ function start() {
   local deployed_config_root=$4
   local cluster_id=$5
   local network_plugin=$6
-  local wait_for_cluster=$7
-  local node_count=$8
-  local additional_args=$9
+  local container_runtime=$7
+  local wait_for_cluster=$8
+  local node_count=$9
+  local additional_args=${10}
 
   # docker-in-docker's use of volumes is not compatible with SELinux
   check-selinux
 
-  echo "Starting dind cluster '${cluster_id}' with plugin '${network_plugin}'"
+  runtime_endpoint=
+  if [[ "${container_runtime}" = "dockershim" ]]; then
+    # dockershim is default and doesn't need an endpoint path
+    runtime_endpoint=
+  elif [[ "${container_runtime}" = "crio" ]]; then
+    runtime_endpoint="/var/run/crio.sock"
+  elif [[ -n "${container_runtime}" ]]; then
```

Comment:
This would leave ${runtime_endpoint} unset if ${container_runtime} was "". Which would only happen if you did `-c ""` anyway, in which case you deserve to lose. Just make this "`else`"

Reply:
done